src/components/link/link.spec.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import 'jest-styled-components';
+import TestRenderer from 'react-test-renderer';
+import { Link as RouterLink } from 'react-router';
+import Link from './link.component';
+import { assertStyleMatch } from '../../__spec_helper__/test-utils';
+
+const render = (props) => {
+  return TestRenderer.create(<Link { ...props }>test</Link>);
+};
+
+describe('Link', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    // eslint-disable-next-line jsx-a11y/anchor-is-valid
```

Comment:
why do you disable eslint for the next line?

Reply:
The above eslint rule force us to make `anchor` with `href` property, but we want to have that without href because we do not know what prop we will need